scripts/5-stake-paymaster.ts

```diff
@@ -0,0 +1,131 @@
+import { ethers } from "hardhat";
+import { DEPLOYMENT_SALTS, isContract } from "./utils";
+import { Deployer, Deployer__factory } from "../typechain-types";
+import { TokenPaymaster__factory } from "@account-abstraction/contracts";
+import { BigNumber } from "ethers";
+import { EntryPoint__factory } from "../account-abstraction/typechain";
+import { formatEther } from "ethers/lib/utils";
+
+const provider = ethers.provider;
+const DEPLOYER_CONTRACT_ADDRESS =
+  process.env.DEPLOYER_CONTRACT_ADDRESS_PROD || "";
+const entrypointAddress =
+  process.env.ENTRY_POINT_ADDRESS ||
+  "0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789";
+const entrypointContract = EntryPoint__factory.connect(
+  entrypointAddress,
+  provider
+);
+
+type PaymasterStakeConfig = {
+  unstakeDelayInSec: number;
+  paymasterStakeInWei: BigNumber;
+};
+
+const paymasterStakeConfig: Record<number, PaymasterStakeConfig> = {
+  80001: {
```

Comment:
add config for more chains with approx staking 100 usd worth native tokens

Reply:
Added for mainnets, but have kept 0.01ETH for testnets